css/obBase.css

```diff
@@ -6400,3 +6400,18 @@ input[type="checkbox"].fieldItem:checked + label .togLabelOff {
 #ov1 .TODO.TODO.TODO.TODO.TODO {
   display: none; /* hide in production */
 }
+
+
+.peersList {
+  margin-top: 10px;
+  display: block;
+  color: #9999A0;
+  max-height: 200px;
+  overflow: auto;
+
+}
+
+.peersList>ul{
+  list-style-type: none;
+  padding-left: 0;
+}
```

Comment:
It would be nice to add some padding or margin to the bottom of each li so they're not so crowded. But we're not asking you to be a designer, I can go in later and fancy things up.